command/sub8_alarm/alarm_handlers/kill.py

```diff
@@ -8,13 +9,23 @@ class Kill(HandlerBase):
     def __init__(self):
         self._killed = False
         self._last_mission_killed = False
+        self.bagging_server = rospy.ServiceProxy('/online_bagger/dump', BaggerCommands)
 
     def raised(self, alarm):
         self._killed = True
+        self.bagger_dump()
 
     def cleared(self, alarm):
         self._killed = False
 
+    def bagger_dump(self):
+        ''' Dump online_bagger'''
```

Comment:
Try to keep your documentation strings in this format:
    '''
    This function does things.
    ''''
    self.do_things()

Not that there are no spaces between the function definition, documentation string, or first line of code. The only reason for spacing here is if the first line after the documentation string is a comment. I believe this is PEP8, but it may just be OCD; someone correct me if I'm wrong.